bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Eclipse/Generator.cpp

```diff
@@ -0,0 +1,117 @@
+/// Apache License 2.0
+
+#include <OpenSpaceToolkit/Astrodynamics/Eclipse/Generator.hpp>
+
+inline void OpenSpaceToolkitAstrodynamicsPy_Eclipse_Generator(pybind11::module& aModule)
+{
+    using namespace pybind11;
+
+    using ostk::core::container::Array;
+    using ostk::core::type::Shared;
+    using ostk::core::type::String;
+
+    using ostk::physics::Environment;
+    using ostk::physics::environment::utilities::Eclipse;
+    using ostk::physics::time::Duration;
+    using ostk::physics::time::Interval;
+
+    using ostk::astrodynamics::eclipse::Generator;
+    using ostk::astrodynamics::Trajectory;
+
+    class_<Generator, Shared<Generator>>(
+        aModule,
+        "Generator",
+        R"doc(
+            An eclipse generator.            
+        )doc"
+    )
+
+        .def(
+            init<const Environment&, const Duration&, const Duration&>(),
+            R"doc(
+                Constructor.
+
+                Args:
+                    environment (Environment): The Environment to use during the search. Defaults to the default environment.
+                    search_step_size (Duration): The step size to use during the search. Defaults to 60 seconds.
+                    search_tolerance (Duration): The tolerance to use during the search. Defaults to 1 millisecond.
```

Comment:
```suggestion
                    environment (Environment): The Environment to use during the search. Defaults to the Environment.default().
                    search_step_size (Duration): The step size to use during the search. Defaults to Duration.seconds(60.0).
                    search_tolerance (Duration): The tolerance to use during the search. Defaults to Duration.milliseconds(1.0).
```
Just to be consistent.